cmobase/background.adoc

```diff
@@ -5,19 +5,19 @@ This chapter provides information common to all CMO extensions.
 
 === Memory and Caches
 
-A _memory location_ is a physical resource in a system uniquely identified by a
-_physical address_. The _observers_ of a given memory location consist of the
-RISC-V harts or I/O devices that can access that memory location. A given
+A _memory location_ is a physical resource in a system uniquely identified by an
+_interconnect physical address_. The _observers_ of a given memory location
```

Comment:
"_**interconnect**_ physical address" is a new concept not mentioned elsewhere in the RISC-V ISA specs, and I wonder whether introducing it here without explanation introduces questions.  Can we please either:
1. Remove the adjective "interconnect".
2. Define how an "interconnect physical address" is different than the existing defined physical address in the ISA.

Reply:
Maybe the term "globally unique" would apply?

Reply:
After talking with @ingallsj, I agree with him that approaching this issue  is more than we want to address right now.  Will  convert this into a TBD item.

Reply:
Per John's suggestion, have removed all of the changes related to "interconnect physical address" and added two TBD items to revisit these issues later.